### PR TITLE
Adding domains field to auto-approval config

### PR DESCRIPTION
**Pull request by dperaza4dustbit**
Implements: [ASC-444](https://issues.redhat.com//browse/ASC-444)

CRD change generated by api `make generate` call.

### CODE REVIEW

File: config/crd/bases/toolchain.dev.openshift.com_toolchainconfigs.yaml [hunk 1]
```diff
@@ -47,42 +47,20 @@ spec:
...
-                          specificPerMemberCluster:
-                            additionalProperties:
-                              type: integer
-                            description: Contains a map of specific capacity thresholds
-                              (in percentage of usage) for particular member clusters
-                              mapped by their names
-                            type: object
-                            x-kubernetes-map-type: atomic
-                        type: object
-                    type: object
```
Comment (dperaza4dustbit) [inline config/crd/bases/toolchain.dev.openshift.com_toolchainconfigs.yaml:85]: This was picked up when I rebased and generated CRD again. Looks to related to changes in this PR: https://github.com/codeready-toolchain/api/pull/411/files

Comment (dperaza4dustbit) [reply config/crd/bases/toolchain.dev.openshift.com_toolchainconfigs.yaml:85]: Looks like this PR needs to be merged before this one then I can rebase: https://github.com/codeready-toolchain/host-operator/pull/1000/files

**Comment by alexeykazakov** [comment]
I've merged https://github.com/codeready-toolchain/host-operator/pull/1000/. So you can re-generate now.

**Review by MatousJobanek — APPROVED** [review]

**Review by alexeykazakov — APPROVED** [review]

**Pull request merged**